### PR TITLE
Fix ownership of the service

### PR DESCRIPTION
**Pull request by mrakitin**
To follow up on our discussion in Slack.

### CODE REVIEW

**Review by danielballan — APPROVED** [review]

File: jupyter-fmx.service [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Description=FMX Jupyter service
 
 [Service]
-User=softioc-fmx
+User=xf17id2
 Group=n2sn-instadmin
```
Comment (mrakitin) [inline jupyter-fmx.service]: @danielballan, should I also change this group to `n2sn-inststaff-fmx`?

Comment (cryos) [reply jupyter-fmx.service]: Why not `xf17id2` for the group too? Maybe it needs that for data access?

Comment (danielballan) [reply jupyter-fmx.service]: In legacy (i.e. for the present): we can grant the operator account data access if needed.
In assets (i.e. in the future): no direct access to data files will be needed or allowed.

Comment (danielballan) [reply jupyter-fmx.service]: So, yes, we should change the group to match, I think.

Comment (mrakitin) [reply jupyter-fmx.service]: Addressed via https://github.com/NSLS-II-FMX/services/pull/2/commits/7cfc6a35a53bc6ce473e4eeadfd85acaeb57ff50.

**Review by danielballan — APPROVED** [review]

**Review by JunAishima — APPROVED** [review]

**Pull request merged**